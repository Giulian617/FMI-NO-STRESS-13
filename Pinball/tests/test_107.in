100 297339763131573261
79 75 4
52 79 3
79 57 5
79 59 9
79 96 4
84 79 8
79 100 2
29 79 7
79 16 10
79 80 7
79 7 4
27 79 3
79 42 7
10 79 3
73 79 3
79 64 3
79 81 1
23 79 5
79 4 2
26 79 9
55 79 9
79 48 1
79 5 7
85 79 5
79 65 10
79 6 6
70 79 1
86 79 9
79 36 3
79 77 6
79 82 1
28 79 2
1 79 7
79 11 10
71 79 6
17 79 10
79 61 10
20 79 3
79 69 2
46 79 10
51 79 8
76 79 6
79 31 2
63 79 1
97 79 4
79 35 6
88 79 3
79 62 4
79 41 8
45 79 7
22 79 10
90 79 5
79 2 2
79 25 8
79 78 3
79 33 4
79 37 9
79 47 8
50 79 9
79 24 6
94 79 5
79 44 8
12 79 7
56 79 2
72 79 1
19 79 5
79 83 7
79 39 1
87 79 5
79 98 4
79 74 10
9 79 6
43 79 7
99 79 6
79 67 8
79 8 6
95 79 8
79 60 5
15 79 4
14 79 4
54 79 2
79 58 2
21 79 2
79 3 8
79 92 2
53 79 10
79 93 5
68 79 10
79 89 2
79 40 5
79 30 1
79 32 3
79 34 5
18 79 8
13 79 5
49 79 4
79 91 3
66 79 8
38 79 10
