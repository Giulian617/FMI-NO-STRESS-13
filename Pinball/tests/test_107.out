275215361
150213388
946858274
150213388
691389003
136692164
752938129
136692164
136692164
54179093
438252045
691389003
534599463
752938129
752938129
438252045
438252045
946858274
534599463
54179093
150213388
438252045
534599463
136692164
946858274
211530849
54179093
150213388
691389003
514367415
150213388
54179093
752938129
534599463
136692164
54179093
211530849
438252045
514367415
534599463
946858274
691389003
691389003
946858274
691389003
438252045
946858274
514367415
752938129
211530849
946858274
54179093
438252045
150213388
211530849
150213388
534599463
150213388
211530849
534599463
438252045
752938129
514367415
54179093
438252045
946858274
946858274
438252045
150213388
514367415
136692164
514367415
54179093
438252045
752938129
136692164
136692164
54179093
691389003
514367415
514367415
691389003
946858274
534599463
211530849
534599463
54179093
150213388
534599463
54179093
150213388
534599463
534599463
946858274
752938129
752938129
752938129
136692164
150213388
